packages/core/src/components/private/BaseInputRange/styles.ts

```diff
@@ -0,0 +1,108 @@
+import { StyleSheet } from '../../../hooks/useStyles';
+import toRGBA from '../../../utils/toRGBA';
+
+export const HANDLE_SIZE = 14;
+export const HALF_HANDLE_SIZE = HANDLE_SIZE / 2;
+export const TRACK_HEIGHT = 4;
+export const MARK_SIZE = TRACK_HEIGHT;
+
+export const stylesheetInputRange: StyleSheet = ({ unit, ui, color, font, pattern }) => ({
+  container: {
+    position: 'relative',
+    height: 3 * unit,
+  },
+
+  container_disabled: {
+    opacity: 0.4,
+  },
+
+  input: {
+    transform: 'translateY(-2px)',
+    '-webkit-appearance': 'none',
```

Comment:
i think `appearance` is valid too? might help to add that as well?

Reply:
removed `-webkit`-prefixed styles entirely, looked the same in chrome + firefox 🤷‍♀️